dist/js/shepherd.js

```
@@ -157,12 +173,33 @@ var Step = (function (_Evented) {
         }
       }
 
-      if (!this.options.buttons) {
+      // Button configuration
+
+      var buttonsJson = JSON.stringify(this.options.buttons);
+      var buttonsAreDefault = isUndefined(buttonsJson) || buttonsJson === "true";
+
+      var buttonsAreEmpty = buttonsJson === "{}" || buttonsJson === "[]" || buttonsJson === "null" || buttonsJson === "false";
+
+      var buttonsAreArray = !buttonsAreDefault && isArray(this.options.buttons);
+
+      var buttonsAreObject = !buttonsAreDefault && isObject(this.options.buttons);
+
+      // Show default button if undefined or 'true'
+      if (buttonsAreDefault) {
         this.options.buttons = [{
           text: 'Next',
-          action: this.tour.next
+          action: this.tour.next,
+          classes: 'btn'
         }];
-      }
+
+        // Can pass in an object which will assume asingle button
+      } else if (!buttonsAreEmpty && buttonsAreObject) {
+          this.options.buttons = [this.options.buttons];
```

Now silently accepts an object and keeps the later `map